app/components/new-signup/checkbox.hbs

```diff
@@ -0,0 +1,31 @@
+<div class='user-details'>

```

Comment:
Is the checkbox a generic component or related to new signup? If it is generic we can move out of new-signup